bootstrap/bootstrap.sh

```diff
@@ -334,8 +345,19 @@ else # whatever may happen else...
 	exit 1
 fi
 
+REQUIRED_ROLES=(
+	"roles/compute.networkAdmin"
+	"roles/compute.securityAdmin"
+	"roles/storage.admin"
+	"roles/storage.objectAdmin"
+	"roles/iam.serviceAccountKeyAdmin"
+	"roles/iam.serviceAccountAdmin"
+	"roles/iam.securityAdmin"
+	"roles/iam.roleAdmin"
+	"roles/serviceusage.serviceUsageAdmin"
+)
 echo "Binding roles to service account '${SA_FULL_NAME}'..." | fold -s -w 80
-for ROLE in roles/compute.networkAdmin roles/compute.securityAdmin roles/storage.admin roles/storage.objectAdmin roles/iam.serviceAccountKeyAdmin roles/iam.serviceAccountAdmin roles/iam.securityAdmin roles/iam.roleAdmin roles/serviceusage.serviceUsageAdmin; do
+for ROLE in "${REQUIRED_ROLES[@]}"; do
 	echo "  * Binding ${ROLE}..."
```

Comment:
It seems that errors created by gcloud invocation (lines 360-383) are not trapped. I understand that the code in line 389 and following will show an error, but we should also print the error created by gcloud. Now we are throwing it away with 2>&1

Reply:
Ok, my bad